### PR TITLE
release(v1.1.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.1.0-alpha.0 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

---
This change is [Reviewable](https://reviewable.io/reviews/siderolabs/talos/5268)

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: api/machine/machine.proto [hunk 1]
```diff
@@ -82,12 +82,12 @@ message ApplyConfigurationRequest {
   bytes data = 1;
   // replaced by mode
   bool on_reboot = 2 [
-    (common.remove_deprecated_field) = "v1.1",
```
Comment (smira) [inline api/machine/machine.proto:85]: bumped the deprecation, as at the very least we need to have it for the upgrade tests, and I think we should drop these pretty important things couple of cycles later

**Pull request merged**